src/commands/refreshSubscriptions.ts

```diff
@@ -0,0 +1,14 @@
+import * as k8s from 'vscode-kubernetes-tools-api';
+import { IActionContext } from "vscode-azureextensionui";
+
+export default async function refreshSubscriptions(context: IActionContext, target: any): Promise<void> {
```

Comment:
This says 'subscriptions' plural - I thought it was refreshing the cluster list for the selected subscription?